src/content/docs/infrastructure/host-integrations/host-integrations-list/nuxt-js-integration.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Nuxt.js integration
+tags:
+  - nuxt.js integration
+  - nuxt integration
+  - new relic integrations
+translate:
+  - jp
+  - kr
+metaDescription: "Learn how to query your New Relic data with NRQL, our SQL-like query language."
+redirects:
+---
+With our Nuxt.js dashboard, you can easily track your application error rate, initial page load average, route changes, throughput, and page views with JavaScript errors. Built with our infrastructure agent and our Vue browser monitoring, the Nuxt.js dashboard lets you view critical information all in one place.
```

Comment:
https://github.com/newrelic/newrelic-quickstarts/pull/1328/files 

Let's add [dashboards/nuxt-js/nuxtjs01.png](https://github.com/newrelic/newrelic-quickstarts/pull/1328/files#diff-8a36e50cb64fa7ba170006fa52fe2b8c4fa422352c5a6c194ce31f166a2da3b7) this image at the top here

Reply:
Thanks for the link to this, I've added it right after the intro